XamarinCommunityToolkitSample.Android/SplashActivity.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Android.App;
+using Android.Content;
+using Android.Content.PM;
+using Android.OS;
+using Android.Support.V7.App;
+
+namespace Xamarin.CommunityToolkit.Sample.Droid
+{
+	[Activity(Label = "XamarinCommunityToolkitSample", Icon = "@mipmap/icon", Theme = "@style/SplashTheme", MainLauncher = true, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation)]
+	public class SplashActivity : AppCompatActivity
+	{
+		protected override void OnCreate(Bundle savedInstanceState)
+		{
+			base.OnCreate(savedInstanceState);
+			var intent = new Intent(this, typeof(MainActivity));
+			StartActivity(intent);
```

Comment:
```StartActivity(typeof(MainActivity));``` ? But ugh...

Reply:
I'll be more terse next time

Reply:
No worries :) That's just my *personal* preferences